RELEASE/scripts/autoscend/quests/level_13.ash

```diff
@@ -469,6 +469,11 @@ boolean L13_towerNSContests()
 				{
 					if(crowd2Insufficient()) fightClubSpa($effect[Ten out of Ten]);
 				}
+
+				if(in_small() && crowd2Insufficient() && have_effect($effect[Piratastic])==0)
+				{
+					auto_wishForEffect($effect[Piratastic]);
```

Comment:
should probably consider pulling these in normal rather than wishing for them and/or using clovers to get them if the user doesn't have the monkey glove or genie.